examples/plot_knockoffs_wisconsin.py

```diff
@@ -0,0 +1,237 @@
+"""
+Controlled multiple variable selection on the Wisconsin breast cancer dataset
+=============================================================================
+
+In this example, we explore the basics of variable selection and illustrate the need to
+statistically control the amount of faslely selected variables. We compare two variable
+selection methods: the Lasso and the Model-X Knockoffs :footcite:t:`candes2018panning`.
+We show how the Lasso is not robust to the presence of irrelevant variables, while the
+Knockoffs (KO) method is able to address this issue.
+"""
+
+import numpy as np
+import pandas as pd
+
+seed = 0
+rng = np.random.RandomState(seed)
+
+
+########################################################################################
+# Load the breast cancer dataset
+# ------------------------------------------------------
+# There are 569 samples and 30 features that correspond to tumor attributes.
+# The downstream task is to classify tumors as benign or malignant. We leave out 10% of
+# the data to evaluate the performance of the Logistic Lasso (Logistic Regression with
+# L1 regularization) on the prediction task.
+from sklearn.datasets import load_breast_cancer
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+
+data = load_breast_cancer()
+X = data.data
+y = data.target
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, test_size=0.1, random_state=seed
+)
+
+scaler = StandardScaler()
+X_train = scaler.fit_transform(X_train)
+X_test = scaler.transform(X_test)
+n_train, p = X_train.shape
+n_test = X_test.shape[0]
+feature_names = [str(name) for name in data.feature_names]
+
+
+#############################################################################
+# Selecting variables with the Lasso
+# ------------------------------------------
+# We want to select variables that are relevant to the outcome, i.e. tumor
+# charateristics that are associated with tumor malignance. We start off by applying a
+# classical method using Lasso logistic regression and retaining variables with non-null
+# coefficients:
+from sklearn.linear_model import LogisticRegressionCV
```

Comment:
Why do you import LogisticRegressionCV and not Lasso?

You should add an short explanation in the text.

Reply:
It is because Lasso is a regression model while the Wisconsin dataset is a binary classification problem. Following the previous comment from @bthirion I modified the text to include: 
 > [...] the Logistic Lasso (Logistic Regression with L1 regularization) on the prediction task.

Reply:
Can you also add it in the section name?

Reply:
done